pkg/api/v1/types.go

```diff
@@ -693,6 +693,10 @@ type VMReplicaSetSpec struct {
 
 	// Template describes the pods that will be created.
 	Template *VMTemplateSpec `json:"template" valid:"required"`
+
+	// Indicates that the replica set is paused.
+	// +optional
+	Paused bool `json:"paused,omitempty" protobuf:"varint,7,opt,name=paused"`
```

Comment:
suspended/resumed is more commonly used terminology for Iaas providers. 

For example, you 'suspend' and 'resume' an AWS autoscaling group. GCE uses the same  terminology as well.

Reply:
hm, kubernetes uses paused on deployments and kubectl uses `kubectl pause` too.

Reply:
wfm then :)